types/index.d.ts

```diff
@@ -175,7 +175,8 @@ export class Rule implements RuleProperties {
   constructor(ruleProps: RuleProperties | string);
   name: string;
   conditions: TopLevelCondition;
-  event: Event;
+  ruleEvent: Event;
+  event: Event
```

Comment:
Last thing, we should mark one of these as deprecated.
No reason to remove it until a v8 but we should be starting that process.

Reply:
So we'll deprecate `ruleEvent`, right? (Because we want a `Rule` to still implement `RuleOptions`)

Reply:
correct.